osmo_jupyter/timezone_test.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+
+import pytest
+import pandas as pd
+
+from osmo_jupyter import timezone as module
+
+
+@pytest.mark.parametrize('input_datetime, expected_iso_output', [
+    ('2018-01-01 12:00', '2018-01-01T12:00:00-08:00'),
+    ('2018-06-01 12:00', '2018-06-01T12:00:00-07:00'),
+])
+def test_to_aware(input_datetime, expected_iso_output):
+    output_datetime = module.to_aware(input_datetime)
+
+    assert isinstance(output_datetime, datetime.datetime)
+    assert output_datetime.isoformat() == expected_iso_output
+
+
+def test_to_aware_blows_up_if_timezone_provided():
```

Comment:
kind of weird to test the functionality of the 3rd party library

Reply:
but I guess if it's functionality we want, maybe it's nice to call that out?

Reply:
yeah I was going to write the test anyway before I realized it was already implemented. should prevent us accidentally removing the functionality